docs/extensions-plugins/cli.mdx

```diff
@@ -0,0 +1,395 @@
+---
+title: Pieces for Developers CLI Tool
```

Comment:
I think in general, just a look at proper formatting should be followed. 

Also we should add some assets - GIFs of using the CLI.

Reply:
Also please remove the Inconsistent capitalization: sometimes you use "Pieces OS" and other times "Pieces os"

And you can remove all the typos / grammatical errors

Reply:
Also if you can follow the structure as recommended by @mason-at-pieces for other products